packages/mc-scripts/src/utils/get-config-diff.spec.js

```diff
@@ -0,0 +1,223 @@
+const getConfigDiff = require('./get-config-diff');
+
+const createOldTestConfig = (customConfig) => ({
+  name: 'test name',
+  description: 'test description',
+  url: 'https://test.com',
+  icon: '<svg><path fill="#000000"></path></svg>',
+  permissions: [],
+  mainMenuLink: {},
+  submenuLinks: [],
+  ...customConfig,
+});
+
+const createNewTestConfig = (customConfig) => ({
+  name: 'test name',
+  description: 'test description',
+  url: 'https://test.com',
+  icon: '<svg><path fill="#000000"></path></svg>',
+  permissions: [],
+  mainMenuLink: {},
+  submenuLinks: [],
+  ...customConfig,
+});
```

Comment:
I guess we only need one function here:

```suggestion
const createTestConfig = (customConfig) => ({
  name: 'test name',
  description: 'test description',
  url: 'https://test.com',
  icon: '<svg><path fill="#000000"></path></svg>',
  permissions: [],
  mainMenuLink: {},
  submenuLinks: [],
  ...customConfig,
});
```

Reply:
Yes only one function

Reply:
ahh.. True.